src/Api/Models/Request/DeviceEncryptedKeyRequestModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Core.Utilities;
+
+namespace Bit.Api.Models.Request;
+
+public class DeviceEncryptedKeyRequestModel
+{
+    [Required]
+    [EncryptedString]
```

Comment:
Do you know where the default is set to validate model states? We have a ton of `ModelState.IsValid` in the Admin project but I'm not sure if those are necessary or not

Reply:
@jlf0dev I actually had to look into this a bit, according to [this](https://learn.microsoft.com/en-us/aspnet/core/mvc/models/validation?view=aspnetcore-7.0#model-state) model validation is automatically on when the controller has `[ApiController]` decorated on it and I expected that to be the case in our controllers but that's not the case actually. 

We get it through applying our [own controller filter](https://github.com/bitwarden/server/blob/5474d3da18d92517e32d4a9e07dab9697eb9cace/src/Api/Utilities/ModelStateValidationFilterAttribute.cs) that checks the model state (through it's inherited class) by itself and returns an error response if it's invalid. That filter is added via this convention that we only add in the `Api` project. 

https://github.com/bitwarden/server/blob/5474d3da18d92517e32d4a9e07dab9697eb9cace/src/Api/Startup.cs#L148

So it seems that the ones in `Admin` are required right now, but there are multiple ways we could make it unnecessary.